src/openrouter-chat-language-model.test.ts

```diff
@@ -488,10 +488,34 @@ describe("doStream", () => {
 
     // note: space moved to last chunk bc of trimming
     expect(await convertReadableStreamToArray(stream)).toStrictEqual([
+      {
+        type: "response-metadata",
+        id: "chatcmpl-96aZqmeDpA9IPD6tACY8djkMsJCMP",
+      },
```

Comment:
@louisgv what was the context behind these additions that were making the tests fail?  some upstream thing sending down new data that they previously didnt when this test was first written?

Reply:
This PR #9  started adding them in one of the PR, but didn't update the test to reflect that